.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,54 +1,31 @@
----
 name: Bug report
 about: Create a report to help us improve
 title: ''
 labels: ''
 assignees: ''
 
----
-
 ### Bug Title:
 Please provide a CONCISE description of the bug.
 
----
 ### Bug Description:
 Please provide a DETAILED description of the bug.
 
----
 ### Reproduction Steps:    
 **1.**    
 **2.** If needed, delete if not    
 **3.** If needed, delete if not
 
----
 ### Expected Behavior:
 What should have happened if the bug didn't occur.
 
----
 ### Actual Behavior:
 What actually happened, and how it differed from the expected behavior.
 
----
 ### Screenshots (If applicable):
 If possible, add screenshots to help explain the problem.
 
----
-### Desktop (please complete the following information):
```

Comment:
I don't think we need these two sections. Right now, PASS is purely a browser application, so the desktop/smartphone fields aren't useful to our bug reports. The only thing that matters is the browser, and at this stage we only support the most recent version of chromium. Any bug report that isn't on the most recent version of chromium is immediately invalid.

If that changes, we can add this section back in with the configurations we support.